src/components/styles/ProjectPreview.styled.ts

```diff
@@ -0,0 +1,63 @@
+import styled, { keyframes, FontType } from 'styled-components'
+import { fontTypeCss } from '../../styles/index.styled'
+
+const FadeIn = keyframes`
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+`
+
+export const PreviewText = styled.div<{ hasAnimation: boolean; fontType: FontType }>`
```

Comment:
in that case consider making hasAnimation optional

Reply:
acb0a0e9ab69f76346e7d80a15addbffef9a7514